Languages/german.lang

```diff
@@ -252,79 +253,79 @@ msgid "Block Categories Modify"
 msgstr "Sperre Kategorie-Änderung"
 
 msgid "Block Cover Downloads"
-msgstr "Blocke Cover-Downloads"
+msgstr "Sperre Cover-Downloads"
 
 msgid "Block Custom Paths"
-msgstr "Blocke eigene Pfade"
+msgstr "Sperre eigene Pfade"
 
 msgid "Block Feature Settings"
-msgstr "Blocke Feature Optionen"
+msgstr "Sperre Feature Optionen"
 
 msgid "Block Game Install"
-msgstr "Blocke Spieleinstallation"
+msgstr "Sperre Spieleinstallation"
 
 msgid "Block Game Settings"
-msgstr "Blocke Spieleoptionen"
+msgstr "Sperre Spieleoptionen"
 
 msgid "Block GameID Change"
-msgstr "Blocke GameID-Änderung"
+msgstr "Sperre GameID-Änderung"
 
 msgid "Block Global Settings"
-msgstr "Sperre alle Einstellungen"
+msgstr "Sperre Einstellungen"
 
 msgid "Block Gui Settings"
-msgstr "Blocke GUI-Optionen"
+msgstr "Sperre GUI-Optionen"
 
 msgid "Block HBC Menu"
-msgstr "Blocke HBC Menü"
+msgstr "Sperre HBC Menü"
 
 msgid "Block Hard Drive Settings"
-msgstr "Blocke Festpl. Optionen"
+msgstr "Sperre Festpl. Optionen"
 
 msgid "Block IOS Reload"
-msgstr "Blocke IOS-Neuladen"
+msgstr "Sperre IOS-Neuladen"
 
 msgid "Block Loader Layout Button"
-msgstr ""
+msgstr "Sperre Loader-Layout Button"
 
 msgid "Block Loader Mode Button"
-msgstr "Blocke Loader-Modus Button"
+msgstr "Sperre Loader-Modus Button"
 
 msgid "Block Loader Settings"
-msgstr "Blocke Loaderoptionen"
+msgstr "Sperre Loaderoptionen"
 
 msgid "Block Parental Settings"
-msgstr "Blocke Altersoptionen"
+msgstr "Sperre Altersoptionen"
 
 msgid "Block Priiloader Override"
-msgstr "Blocke Priiloader Override"
+msgstr "Blocke Priiloader Überschrei."
 
 msgid "Block Reset Settings"
-msgstr "Blocke Resetoptionen"
+msgstr "Sperre Resetoptionen"
 
 msgid "Block SD Reload Button"
-msgstr "Blocke SD Reload Button"
+msgstr "Sperre SD-Neuladen"
 
 msgid "Block Sound Settings"
-msgstr "Blocke Soundoptionen"
+msgstr "Sperre Soundoptionen"
 
 msgid "Block Theme Downloader"
-msgstr "Blocke Theme Downloader"
+msgstr "Sperre Theme Downloader"
 
 msgid "Block Theme Menu"
-msgstr "Blocke Theme Menü"
+msgstr "Sperre Theme Menü"
 
 msgid "Block Title Launcher"
-msgstr "Blocke Title Launcher"
+msgstr "Sperre Channel Launcher"
 
 msgid "Block Updates"
-msgstr "Blocke Updates"
+msgstr "Sperre Updates"
 
 msgid "Boot Content"
 msgstr "Boote Inhalt"
 
 msgid "Boot Neek System Menu"
-msgstr ""
+msgstr "Boote Neek System Menü"
```

Comment:
` Boot` is not German.

Reply:
"Boote" was used in the original German translation to keep the string size as short as possible. "Starte" would also be a correct translation. "Booten" is also Germanized and can also be found in the Duden ([here](https://www.duden.de/rechtschreibung/booten)).